woothemes/boutique
# 👌 IMPROVE: Update tooling and dependencies (#70)

Fixes #70

Update tooling and dependencies

### How to test the changes in this Pull Request:

1. Checkout GitHub repo
2. Run `npm i` → All dependencies should be up to date
3. Run `npm run build` → An updated build can be created without problems
4. Run `npm run css` → The file https://github.com/woocommerce/boutique/blob/master/style.scss can be converted into  https://github.com/woocommerce/boutique/blob/master/style.css and https://github.com/woocommerce/boutique/blob/master/style-rtl.css without problems
4. Run `npm run css` and apply a CSS change to https://github.com/woocommerce/boutique/blob/master/style.scss → New versions of the files https://github.com/woocommerce/boutique/blob/master/style.css and https://github.com/woocommerce/boutique/blob/master/style-rtl.css are rendered without problems

### Changelog

> Tweak - Update software dependencies. #70
